packages/contracts/scripts/shared/constants/addresses.ts

```diff
@@ -0,0 +1,57 @@
+export const DEPLOYMENT_OVERRIDES: Record<string, any> = {
+    mainnet: {
+        deployer: "0xefC0e701A824943b469a694aC564Aa1efF7Ab7dd",
+        whaleAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        tester: "0x89eae71b865a2a39cba62060ab1b40bbffae5b0d",
+        sablier: "0xA4fc358455Febe425536fd1878bE67FfDBDEC59a",
+        DAI: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+        USDC: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+        USDT: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+        curve3CrvBasePool: "0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7",
+        curve3CrvToken: "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490",
+        curveFactory: "0x0959158b6040D32d04c301A72CBFD6b39E21c9AE",
+        curveWhaleAddress: "0xC2872ab688400940d5a6041599b3F7de20730d49",
+        daiWhaleAddress: "0x16463c0fdB6BA9618909F5b120ea1581618C1b9E",
+        sushiMultiSig: "0x9a8541Ddf3a932a9A922B607e9CF7301f1d47bD1",
+        UbqWhaleAddress: "0xa53A6fE2d8Ad977aD926C485343Ba39f32D3A3F6",
+        MasterChefAddress: "0x8fFCf9899738e4633A721904609ffCa0a2C44f3D",
+        MetaPoolAddress: "0x20955cb69ae1515962177d164dfc9522feef567e",
+        BondingAddress: "0x831e3674Abc73d7A3e9d8a9400AF2301c32cEF0C",
+        BondingV2Address: "0xdAC17F958D2ee523a2206206994597C13D831ec7", // FAKE ADDRESS, TO BE REPLACED AFTER V2 DEPLOYMENT
```

Comment:
These go to Tether's contract. We never updated this address in a long time, which makes me wonder if we actually need this file? Don't we have a json fixture somewhere already with all the important addresses?

Reply:
removed
https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/399a6457f309087721e3b88689ac8bd4fb3af2ed